### PR TITLE
clarify image resolution guidelines

### PR DESCRIPTION
**Pull request by markwkidd**

### CODE REVIEW

**Comment by alfrix** [comment]
i don't agree with 
`Images with native width 256px and 512px should be scaled down to 256px wide`

**Comment by markwkidd** [comment]
I will be glad to change that information to whatever is correct. What should I put?

Those guidelines are what I heard from @RobLoach

**Comment by alfrix** [comment]
@RobLoach the whole reason of no wider than 512px, is to keep repo sizes under control, downscaling images to 256 really hurt the quality, test at 4k you will see it
i support the original rule of no wider than 512px, if smaller just upload as is.

File: README.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ sh update_modules.sh
     &*/:`<>?\|"
     ```
 - Images must be `.png` format
-- Images should be no wider than 512px
+- Image resolution guidelines:
+  - Images with native width larger than 512px should be scaled down to 512px wide
+  - Images with native width of 512px should be added as-is
+  - Images with native width 256px and 512px should be scaled down to 256px wide
```
Comment (RobLoach) [inline README.md]: Should this be "Images with native width **between** 256px and 512px..."?

Comment (markwkidd) [reply README.md]: You're right. Fixed, thanks!

**Comment by RobLoach** [comment]
@alfrix What condition would you change on here?

**Comment by Europia79** [comment]
I vote NO for having to down-scale (for example) 483 pixel to 256 pixels.  But after browsing the current state of the repository, I am left with questions.  So perhaps clarification and/or rule changes are in order ?

As an example, look at some of the box-art for the SNES.  They all have a width of 512 pixels.  But the Japanese boxes are vertically endowed (book shaped) with an average height of 930 pixels, whereas the North American boxes are vertically challenged (but horizontally endowed - shaped like wide-screens)  with an average height of only 374 pixels.  Such a large disparity in resolution is a little weird.  Why is that ?

Those Japanese boxes are taking up two to three times the disk space compared to the NA boxes.
Are the Jap boxes too big ?  Or are the NA boxes too small ? (in resolution)

If the NA boxes are too small, then perhaps the rule should be max 512px in height for horizontal boxes, and max 512px in width for vertical boxes.

If the Jap boxes are too big, then perhaps the rule should be max 512px in height for vertical boxes and max 512px in width for horizontal boxes.

Either way, it sucks for Genesis fans (like me) because the normal resolution of a Genesis box is 483x680 which is great rez, but it also takes up double the disk space as even the largest SNES Jap box and over five times the space as those smaller NA boxes.

The `SNES/Named_Boxarts` folder is 517 MB with 3,333 files.
The `Gen/Named_Boxarts` folder is 924 MB with 1,935 files.

So, the problem at hand seems to be balancing image resolution and file size ?
Maybe the repository should be composed of large files at high rez, and then just have the user decide what resolution he/she wants to save as ?

Also, rules should be clear, but hopefully you guys can reach a consensus and close this issue because it gives the perception that submitting a pull request could potentially be a waste of time  (as you might submit something that is in violation of the standards).  Thanks!

**Comment by RobLoach** [comment]
> I vote NO for having to down-scale (for example) 483 pixel to 256 pixels

I think that's reasonable. May be beneficial to keep it simple, stupid...

```
if image.width > 512px then
  resize to 512px width
else
  Leave as is
endif
```

**Pull request merged**